packages/compose/src/hooks/use-resize-observer/_legacy/index.tsx

```diff
@@ -6,14 +6,13 @@ import type { ReactElement } from 'react';
 /**
  * WordPress dependencies
  */
-import {
-	useCallback,
-	useLayoutEffect,
-	useRef,
-	useState,
-} from '@wordpress/element';
+import { useCallback, useRef, useState } from '@wordpress/element';
+/**
+ * Internal dependencies
+ */
+import useResizeObserver from '../index';
```

Comment:
Here we're introducing a circular dependency where `index` imports `_legacy/index` and vice versa. Can we get rid of it? Sometimes it backfires in confusing ways.

Reply:
Done https://github.com/WordPress/gutenberg/pull/65389/commits/35153bfb033ed4d147015a13e0867db8e3622ec2